commodore/dependency_mgmt.py

```diff
@@ -96,21 +96,13 @@ def _read_component_urls(cfg: Config, component_names) -> Dict[str, str]:
             f"Could not read Commodore configuration: {e}"
         ) from e
 
-    for component_name in component_names:
-        component_urls[
-            component_name
-        ] = f"{cfg.default_component_base}/{component_name}.git"
-
-    if commodore_config is None:
-        click.secho("Empty Commodore config file", fg="yellow")
```

Comment:
This should probably be promoted to an error instead of removing the warning.

Reply:
It is not removed. It is now an exception. What should be done differently?

Reply:
I don't see an exception for "the file exists, but is empty". Although I guess the final check which throws an exception if there's no URL for a component kinda covers the case where the file exists but is empty.

Reply:
Yesterday I was pretty done when writing that comment. Now I get your concern.

Yes, the case of an empty file is no longer covered. I removed it as it does not provide much value (if any). The exception for missing URLs is enough.